packages/ppx_import/ppx_import.1.10.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "A syntax extension for importing declarations from interface files"
+maintainer: "whitequark <whitequark@whitequark.org>"
+authors: [ "whitequark <whitequark@whitequark.org>" ]
+homepage: "https://github.com/ocaml-ppx/ppx_import"
+doc: "https://ocaml-ppx.github.io/ppx_import/"
+license: "MIT"
+bug-reports: "https://github.com/ocaml-ppx/ppx_import/issues"
+dev-repo: "git+https://github.com/ocaml-ppx/ppx_import.git"
+tags: [ "syntax" ]
+
+depends: [
+  (
+  "ocaml"                   {>= "4.05.0" &  < "4.10.0"  }
+  "dune"                    {              >= "1.11.0"  }
+  "ppxlib"                  {              >= "0.26.0"  }
+  "ounit"                   { with-test                 }
+  "ppx_deriving"            { with-test  & >= "4.2.1"   }
+  )
+  |
+  (
+  "ocaml"                   { >= "4.10.0"               }
+  "ppx_sexp_conv"           { with-test  & >= "v0.13.0" }
+  "dune"                    {              >= "1.11.0"  }
+  "ppxlib"                  {              >= "0.26.0"  }
+  "ounit"                   { with-test                 }
+  "ppx_deriving"            { with-test  & >= "4.2.1"   }
+  )
```

Comment:
I’m not sure to understand why the dependencies has to be so complicated. Both sides of the disjunction is basically the same except for `ppx_sexp_conv`

This can be simplified into:
```suggestion
  "ocaml"                  {>= "4.05.0" &  < "4.10.0"  } | ("ocaml" {>= "4.10.0"} "ppx_sexp_conv" {with-test & >= "v0.13.0"})
  "dune"                    {              >= "1.11.0"  }
  "ppxlib"                  {              >= "0.26.0"  }
  "ounit"                   { with-test                 }
  "ppx_deriving"            { with-test  & >= "4.2.1"   }
```

Reply:
I’m also unsure why avoiding installing ppx_sexp_conv (especially with such broad constraints) in the first place.
Looking at https://github.com/ocaml-ppx/ppx_import/pull/69 it seems to be because it’s not used but in this case it wouldn’t really matter if it’s installed or not and would simplify everything.

Could we remove the disjunction all together?

Reply:
```suggestion
  "ocaml"                   {              >= "4.05.0"  }
  "dune"                    {              >= "1.11.0"  }
  "ppxlib"                  {              >= "0.26.0"  }
  "ounit"                   { with-test                 }
  "ppx_deriving"            { with-test  & >= "4.2.1"   }
  "ppx_sexp_conv"           { with-test  & >= "v0.13.0" }
```

Reply:
> I’m also unsure why avoiding installing ppx_sexp_conv (especially with such broad constraints) in the first place. Looking at [ocaml-ppx/ppx_import#69](https://github.com/ocaml-ppx/ppx_import/pull/69) it seems to be because it’s not used but in this case it wouldn’t really matter if it’s installed or not and would simplify everything.
> 
> Could we remove the disjunction all together?

I'm afraid the constraint is needed for ppx_import test-suite, as it is not possible to install ppxlib gt 0.26 than ppx_sexp_conv in OCaml 4.09 or less

Reply:
ah I understand the issue now. Fair enough. I’ll commit the simplified disjunction instead then